From fd34fd7a3ec71bde942cdcfc0e24fd911ff5934e Mon Sep 17 00:00:00 2001
From: Mauro Stettler <mauro.stettler@gmail.com>
Date: Sun, 25 Jan 2015 20:26:17 +0900
Subject: [PATCH 161/193] staging: flexfb: Remove initialization to 0

(the initial post had a typo in the short summary, reposting)

This fixes some of the code style issues in the flexfb driver by removing
unnecessary initializations of static variables to 0 or to NULL.

Signed-off-by: Mauro Stettler <mauro.stettler@gmail.com>
Cc: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Cc: Noralf Tronnes <notro@tronnes.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/fbtft/flexfb.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/drivers/staging/fbtft/flexfb.c b/drivers/staging/fbtft/flexfb.c
index 45574a0..4eec888 100644
--- a/drivers/staging/fbtft/flexfb.c
+++ b/drivers/staging/fbtft/flexfb.c
@@ -31,24 +31,24 @@
 #define DRVNAME	    "flexfb"
 
 
-static char *chip = NULL;
+static char *chip;
 module_param(chip, charp, 0);
 MODULE_PARM_DESC(chip, "LCD controller");
 
-static unsigned int width = 0;
+static unsigned int width;
 module_param(width, uint, 0);
 MODULE_PARM_DESC(width, "Display width");
 
-static unsigned int height = 0;
+static unsigned int height;
 module_param(height, uint, 0);
 MODULE_PARM_DESC(height, "Display height");
 
 static int init[512];
-static int init_num = 0;
+static int init_num;
 module_param_array(init, int, &init_num, 0);
 MODULE_PARM_DESC(init, "Init sequence");
 
-static unsigned int setaddrwin = 0;
+static unsigned int setaddrwin;
 module_param(setaddrwin, uint, 0);
 MODULE_PARM_DESC(setaddrwin, "Which set_addr_win() implementation to use");
 
@@ -60,17 +60,17 @@ static unsigned int regwidth = 8;
 module_param(regwidth, uint, 0);
 MODULE_PARM_DESC(regwidth, "Width of controller register (default: 8)");
 
-static bool nobacklight = false;
+static bool nobacklight;
 module_param(nobacklight, bool, 0);
 MODULE_PARM_DESC(nobacklight, "Turn off backlight functionality.");
 
-static bool latched = false;
+static bool latched;
 module_param(latched, bool, 0);
 MODULE_PARM_DESC(latched, "Use with latched 16-bit databus");
 
 
-static int *initp = NULL;
-static int initp_num = 0;
+static int *initp;
+static int initp_num;
 
 /* default init sequences */
 static int st7735r_init[] = { \
-- 
2.1.4

